test/podFailover/controller/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/aks/fundamental/base-ubuntu:v0.0.5
+RUN apt-get update && apt-get install -y util-linux e2fsprogs mount ca-certificates udev xfsprogs
```

Comment:
We do not need these packages installed since the controller and workload programs don't depend on them.